helmfile/cloud-sdk/helmfile.yaml

```diff
@@ -507,8 +507,7 @@ releases:
           cpu: 1500m
           memory: 4Gi
       heapSize: "2048m"
-      readinessProbe:
-        timeoutSeconds: 30
+      readinessProbe: null
```

Comment:
Not sure if this will help, probably it will be merged with default values and default values will be used. Try applying it, I don't have anything against this

Reply:
Diff looks like it would work (https://github.com/Telecominfraproject/wlan-toolsmith/pull/232/checks#step:7:194), but I'll try.